src/Core/Models/Data/Provider/ProviderOrganizationOrganizationDetails.cs

```diff
@@ -10,5 +10,8 @@ public class ProviderOrganizationOrganizationDetails
         public string Settings { get; set; }
         public DateTime CreationDate { get; set; }
         public DateTime RevisionDate { get; set; }
+        public int UserCount { get; set; }
```

Comment:
Any specific reason why this `UserCount` is not nullable while the `ProviderOrganizationResponseModel.UserCount` is?

Reply:
They should both be not nullable, I've changed `ProviderOrganizationResponseModel.UserCount`.